code/style.css

```diff
@@ -1,7 +1,384 @@
-/* the + sign is used to select the sibling that comes immediately next to the element you selected.
-in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
+body {
+  font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+  margin: 0;
+}
+
+/*** HEADER ***/
+
+header {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  color: white;
+  margin: 0;
+  width: 100%;
+  height: 400px;
+
+}
+
+.video-container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: absolute;
+  width: 100%;
+  height: 400px;
+  overflow: hidden;
+}
+
+
+.header-content {
+  position: relative;
+}
+
+header h2 {
+  color: white;
+}
+
+h1 {
+  font-size: 50px;
+  text-align: center;
+  margin: 0px;
+}
+
+h2 {
+  font-size: 25px;
+  text-align: center;
+  color: lightsalmon;
+}
+
+p {
+  margin: 0;
+}
+
+/*** MAIN ***/
+
+main {
+  display: grid;
+  grid-template-rows: repeat(4, 1fr);
+  align-items: center;
+  justify-content: center;
+  
+}
```

Comment:
I love to see that you have given grid a try, mastering both grid and flexbox will give you endless styling possibilities. Great job 🏅